### PR TITLE
Compose fixes in dark mode & themes

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
[ANDROID-10047](https://jira.tid.es/browse/ANDROID-10047) Small fixes regarding the catalog app when using dark mode.

### :construction: How do we do it?
Providing a default MaterialTheme that uses primary, secondary, background and error colors taken from Mística

### ☑️ Checks
- [ ] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [ ] Tested with dark mode.
- [ ] Tested with API 21.

### :test_tube: How can I test this?

https://user-images.githubusercontent.com/4595241/141793125-74dba449-3e63-4ae2-8113-521bd1433082.mp4

### CODE REVIEW

File: library-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Jetpack Compose support
+
+[Jetpack compose](https://developer.android.com/jetpack/compose?gclid=Cj0KCQiAsqOMBhDFARIsAFBTN3fVBPwn0qPY2gE7Ghqy3N3sT1cZa2CQ4luSD5VgiCRZ5UHNQN0Ao2gaAmPrEALw_wcB&gclsrc=aw.ds) is a toolkit to develop native UI in Android. Newer and more powerful than Android Views it has became the default system in newer Telefónica Android applications.
```
Comment (yamal-alm) [inline library-compose/README.md:3]: very minor: can the url be simplified to just https://developer.android.com/jetpack/compose? it looks like some query metadata params has been appended to the url

File: catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/CatalogMainActivity.kt [hunk 2]
```diff
@@ -45,11 +47,12 @@ import com.telefonica.mistica.compose.theme.brand.MovistarProminentBrand
 
 class CatalogMainActivity : ComponentActivity() {
 
-    private var brand by mutableStateOf<Brand>(MovistarBrand)
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
+            var brand by mutableStateOf<Brand>(MovistarBrand)
+            Log.d("gmerinoTest", "la brand ${brand.lightColors.brand}")
```
Comment (yamal-alm) [inline catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/CatalogMainActivity.kt:55]: forgotten line?

**Review by yamal-alm — APPROVED** [review]
nice 👏  thanks!

File: library-compose/README.md (hunk 1 above)
Comment (gmerinojimenez) [reply library-compose/README.md]: Totally, good point :)

File: catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/CatalogMainActivity.kt (hunk 2 above)
Comment (gmerinojimenez) [reply catalog-compose/src/main/java/com/telefonica/mistica/compose/catalog/ui/CatalogMainActivity.kt]: removed, thx :)

File: library-compose/src/main/java/com/telefonica/mistica/compose/theme/MisticaTheme.kt [hunk 3]
```diff
@@ -52,20 +52,18 @@ fun MisticaTheme(
...
-                secondaryVariant = Color.Unspecified,
+            colors = if (darkTheme) {
+                darkColors()
+            } else {
+                lightColors()
+            }.copy(
+                primary = LocalMisticaColors.current.brand,
+                primaryVariant = LocalMisticaColors.current.brand,
+                secondary = LocalMisticaColors.current.brand,
+                secondaryVariant = LocalMisticaColors.current.brand,
```
Comment (dpastor) [inline library-compose/src/main/java/com/telefonica/mistica/compose/theme/MisticaTheme.kt:63]: Ok, we can modify them afterwards if needed

**Review by dpastor — APPROVED** [review]
Nice

**Pull request merged**